src/main/java/org/opentripplanner/netex/mapping/support/NetexMapperIndexes.java

```diff
@@ -0,0 +1,104 @@
+package org.opentripplanner.netex.mapping.support;
+
+
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.Multimap;
+import org.opentripplanner.model.Station;
+import org.opentripplanner.model.StopTime;
+import org.opentripplanner.netex.index.api.NetexEntityIndexReadOnlyView;
+import org.opentripplanner.netex.index.api.ReadOnlyHierarchicalMapById;
+import org.rutebanken.netex.model.DatedServiceJourney;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * The responsibility of this class is to cache and index NeTEx entities uses be more than one mapper.
```

Comment:
I think you want to re-check this sentence. I believe there are some words missing.